### DISCUSSION TITLE
When image selected and saved it shows 'obj'

### DISCUSSION THREAD

**Discussion by MaheshManoharan** [singerdmx/flutter-quill#399] [General]
Hi, 

May be this is not an issue. But I am a beginner. I don't know where to ask. When I selected and saved image in database. 
`
    String content = jsonEncode(
      _controller.document.toDelta().toJson(),
    );`

I use this code to get the content. But when I displayed it in screen I got this.

<img width="196" alt="quill issue" src="https://user-images.githubusercontent.com/69787389/133035231-0ddca6c8-11e9-42ab-922d-ba74fb0d02f9.png">

I dont' want this. I just want to be like in evernote. please help me to remove obj and show nothing in text.

**Comment by li3317** [comment]
Hello, may I know how you are using the content string to display the result? We handle image differently from how we handle other text.

**Reply by MaheshManoharan** [reply]
content is from database.
`    var contentjson = jsonDecode(content.toString());
    Document document = Document.fromJson(contentjson);
    content2 = document.toPlainText();
`

Text(content2.toString()) 

This is how I am used to display the content.
